<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMatchingStatisticsComponent</class>
 <widget class="QFrame" name="CMatchingStatisticsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matching statistics</string>
  </property>
  <layout class="QVBoxLayout" name="vl_MatchingStatistics">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="swift::gui::views::CMatchingStatisticsView" name="tvp_MatchingStatistics">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_Bottom">
     <layout class="QHBoxLayout" name="hl_Bottom">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_MissingOnly">
        <property name="text">
         <string>missing only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_All">
        <property name="text">
         <string>all</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Bottom">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Generate">
        <property name="text">
         <string>generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CMatchingStatisticsView</class>
   <extends>QTableView</extends>
   <header>gui/views/matchingstatisticsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
